### ISSUE TITLE
whats' the AU's meaning?

### ISSUE THREAD

**Issue by hualan** [TadasBaltrusaitis/CLM-framework#37]
hi, could u tell whats' meaning of the AU, tks!

**Comment by AshwinRajendraprasad** [comment]
Action Units. Look up Candide3 model.

**Comment by TadasBaltrusaitis** [comment]
Yes they are called Action Units and are a way of describing facial expressions. You can find more information about them here:
http://www.cs.cmu.edu/~face/facs.htm
https://en.wikipedia.org/wiki/Facial_Action_Coding_System

**Comment by pjyazdian** [comment]
AU means action units which shows the changes in face. there is 46 AUs in face which acts different role in facial expressions. for instance, AU4 (Cheek Raiser) and AU12 (Lip Corner Puller) together act as showing happiness in face.
